api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -712,6 +719,9 @@ def pick_up_tip(
         assert tiprack.is_tiprack, "{} is not a tiprack".format(str(tiprack))
         validate_tiprack(self.name, tiprack, logger)
 
+        if prep_after is None:
+            prep_after = (self.api_version > APIVersion(2, 12))
+
```

Comment:
Strictly, we should also forbid people from overriding `prep_after` when their `apiLevel` is lower than 2.13.

Something like this:

```suggestion
        prep_after_added_in = APIVersion(2, 13)
        if prep_after is None:
            prep_after = (self.api_version >= prep_after_added_in)
        elif self._api_version < prep_after_added_in:
            raise APIVersionError(
                f"prep_after is only available in API {prep_after_added_in} and newer,"
                f" but you are using API {self._api_version}."
            )
```

So that if I write a protocol for my robot that uses `prep_after=something`, and then share it with someone who's using outdated software, they get a hopefully helpful API version error instead of a surprise `TypeError`.